lifelines/plotting.py

```diff
@@ -527,7 +635,7 @@ def iloc(x, i):
         ax.set_yticks(range(0, N))
         ax.set_yticklabels(durations.index)
     else:
-        ax.set_yticks([])
+        ax.yaxis.set_major_locator(MaxNLocator(integer=True))
```

Comment:
Can you explain what `MaxNLocator` does?

Reply:
It just ensures that any y-axis labels are at integer values, otherwise if durations has just a few values (seems to happen with about 20 or less):
```
from lifelines.plotting import plot_lifetimes
import numpy as np
a = np.asarray([20.0, 53.0, 50.0])
plot_lifetimes(a)
```
it results in something like this:
![plot1](https://user-images.githubusercontent.com/19431229/80843002-0aa72600-8bfb-11ea-8373-6743ddaf4a79.png)
instead of:
![plot2](https://user-images.githubusercontent.com/19431229/80843125-62de2800-8bfb-11ea-9b75-916b709fb879.png)